### ISSUE TITLE
Add a HOW YOU CAN HELP US HELP YOU section

### ISSUE THREAD

**Issue by jgclark** [chineham-repair-cafe/site-production#2]
I think we can borrow from the [Restarters page template](https://therestartproject.org/parties/chineham-christ-church-6/) to make a new section to go before SUPPORTERS.

Something like this section heading:

HOW YOU CAN HELP US HELP YOU
Before you come to a Repair Cafe session, please look through these tips:

then the DO's and DONT's in two columns:

Do
- **Check if your device is still within warranty**, you may be eligible for a free repair, replacement or a refund. ([Which?](https://www.which.co.uk/) helps in the UK).
- **Get in touch with organisers if you have any questions** about whether a fix is possible.
- **Wash** any clothes or fabrics you'd like repaired.
- For computers/tablets/phones, take responsibility for your device and your data. We can’t be responsible for your cat pics, so please **back up your device** before you come. (Of course you already do this once a month, right?)
- **Bring all leads, and charge up your device** (if relevant).

Don't
- **Assume repairs will be successful**. We help fix over half of devices at our events. Even if your repair is unsuccessful, you will learn about your device and why it failed.
- **Expect us to have spare parts**. But we can help you source them. Often people visit twice: first we’ll help diagnose the fault, then you source the spare part, and on your return we’ll help you do a replacement.
- **Attempt a mobile screen repair just to save money**. DIY mobile screen repair is a learning opportunity first and foremost. Please see our [advice page](https://therestartproject.org/self-repair/screen-repairs/) first.
- **Bring a microwave**. There are some other items we might decide are not safe to attempt to repair.

**Comment by sibo999** [comment]
The other cafe's have added this as a sepaerate document, 
The DOs and DON’Ts for Chineham Repair Café
The DOs
• Make sure your appliance is in a clean state so our repairer don’t have to clean it before repairing it. Example – grass cuttings removed around mower blades, toast bits removed from base of toaster, clean filter and empty bag/bin on a vacuum cleaner.
• Check if your device is still within warranty. If it is, you may be eligible for a free repair, replacement or a refund. If it is out of warranty, then we’d be happy to look at it for you. Any work we do may invalidate a warranty.
• Get in touch with us if you have any questions about whether a fix is possible. We can be reached via our website or info@chinehamrepair.org.uk
• Take responsibility for your device and your data. Although our volunteers are here to repair your item, do learn from them so you know how to fix your item if the same problem again.
• Backup your data before you bring your device in for repair. (You should regularly back up your devices anyway).
• Charge up your electrical device and bring the electric cable or charger with you. We need to PA test all electrical appliances /devices before and after repair.
• Come back to pick up your item. If our repairer has taken your item home to fix, then pick it up as agreed.
• Leave a donation if you can afford it. Although your repair is free, there are costs involved in running a repair café. You can donate in cash or by card. Thank you for your support.
The DON’Ts
• Assume repairs will always be successful. We will do our best to fix your items. Even if your repair is unsuccessful, you will learn about your item and why it failed. Sometimes, items are really beyond repair.
• Expect us to completely alter a garment for you, for example – reducing a dress from a size 14 to a size 12. It is not something we can do in the time allocated to the repair café. We would happily recommend a local dress maker to do that for you.
• Expect us to have spare parts. We can help you source them. Often customers visit twice: first we’ll help diagnose the fault, then they source the spare part, and on their return we’ll replace their faulty component. Or bring the spare part with you.
• Bring in items that we are not allowed to fix as they will be turned away. These items are mentioned in our website. If unsure, do contact us to check.
• Expect us to dispose of your item which is beyond repair. Please take your item home with you to dispose, and follow any disposal guidelines e.g. WEEE for any electrical items.

**Comment by shymega** [comment]
Yeah, I think if we try and fit all this in on *one single page*, it'll become too saturated for prospective visitors. Maybe we should look into, long-term, a multi-page website structure for the finer details?

**Comment by jgclark** [comment]
This content is much more important than some of the other existing content on the page. We need to make space for it. On the restarters site it is no more than one standard screen full. We currently have about 8 screen fulls.
Specifically this is more important than MISSION / PLAN / VISION section.

**Comment by shymega** [comment]
Agreed. What are your thoughts on a multi-page structure? I think the existing page is a bit too cluttered, or perhaps we could use a modal approach, where some details are hidden on first-load and can be opened using a hamburger menu?

**Comment by jgclark** [comment]
I would naturally have opted for multi-page, but I take Simon's point
about now defaulting to single page (assuming lazy loading of assets) to
suit mobile-first. I think the VISION/MISSION/etc. stuff could go to a
separate page that talks more specifically about vision for expansion in
the Basingstoke area. And replace it on the Chineham-specific page with the
HELPING YOU section.

**Comment by sibo999** [comment]
Just having it a s a pdf document on the website seems to work for others. If you're using a ticketing system you can add details onto electronic tickets.
